lib/components/episode/episode_info.dart

```diff
@@ -92,4 +115,14 @@ class EpisodeInfo extends StatelessWidget {
       ),
     );
   }
+
+  toggleInMyList(WidgetRef ref, ValueNotifier inMyList) {
```

Comment:
It's cleaner to put this function in build() instead of passing the parameters here.

Reply:
and add "void"
```dart
void toggleInMyList() { 
  // ...
}
```